common/ManagementTestShared/Redesign/ManagementRecordedTestBase.cs

```diff
@@ -62,7 +62,8 @@ protected ArmClient GetArmClient(ArmClientOptions clientOptions = default)
             return CreateClient<ArmClient>(
                 TestEnvironment.SubscriptionId,
                 TestEnvironment.Credential,
-                options);
+                options,
+                null);
```

Comment:
Prefer if this goes on the ArmClientOptions given that the majority of this the time this isn't needed.

Reply:
`Activator.CreateInstance()` will nedd full parameter group to get the ArmClient